### PR TITLE
Dev - Fixed the loss of fort data

### PR DESCRIPTION
**Pull request by Abraxas000**
Short Description:  This fixes loss of fort data when the server returns no fort data. 

It seems like the server is saving cycles/bandwidth by only returning fort data when there has been a change to forts in your cells. When it returns no forts, we currently replace our existing data with the empty data.  This fixes that so we keep our fort data if the server returned no fort data.  Wild/lured pokemon are still updated in any case.

Fixes:
- If the server returns no fort data in the cell data, keep our existing fort data. (Web location data is still going to lose fort data, but the bot itself won't)
- I removed duplicate code
- Edited cluster log function to round to nearest meter

Known related Issues:
-https://github.com/PokemonGoF/PokemonGo-Bot/issues/2184

### CODE REVIEW

File: pokemongo_bot/__init__.py [hunk 1]
```diff
@@ -94,19 +94,30 @@ def get_meta_cell(self):
         forts = []
         wild_pokemons = []
         catchable_pokemons = []
+        fortspresent = False
```
Comment (TheSavior) [inline pokemongo_bot/__init__.py]: Do you really need this flag? Could we just check for `len(forts)`?

File: pokemongo_bot/cell_workers/follow_cluster.py [hunk 2]
```diff
@@ -42,7 +42,7 @@ def work(self):
 
                 log_str = log_lure_avail_str + 'Move to destiny. ' + str(cnt) + ' ' + log_lured_str + \
                           'pokestops will be in range of ' + str(self.radius) + 'm. Arrive in ' \
-                          + str(distance(self.bot.position[0], self.bot.position[1], lat, lng)) + 'm.'
+                          + str(round(distance(self.bot.position[0], self.bot.position[1], lat, lng))) + 'm.'
```
Comment (TheSavior) [inline pokemongo_bot/cell_workers/follow_cluster.py]: This change doesn't seem relevant to this PR

**Comment by douglascamata** [comment]
Also fix conflicts with dev branch, please.

File: pokemongo_bot/cell_workers/follow_cluster.py (hunk 2 above)
Comment (Abraxas000) [inline pokemongo_bot/cell_workers/follow_cluster.py]: Agreed, I'm not sure how to separate the commits into different PRs yet. Feel free to drop this commit from the PR.

Comment (TheSavior) [inline pokemongo_bot/cell_workers/follow_cluster.py]: The commits would need to be on different branches. It is a lot of work for us to split up your PR. Since this is such a small change it would be easiest for you to just add a new commit undoing this change, and then on a new branch adding it again.

**Comment by matheussampaio** [comment]
@Abraxas000 One way to separate commits is (or remove commits from that branch) is using `git rebase -i dev` and change `pick` to `drop`.

**Comment by Abraxas000** [comment]
Thanks all for the assistance! I'm used to using TFS so I'm still bumbling my way through git processes.

@TheSavior The cluster change has been reverted and the conflict from the previous merge is resolved.

**Comment by Abraxas000** [comment]
This PR got very messy. I'm starting with a fresh fork and branching my edits so I won't have to deal with this again later. I'm sure there's a more elegant way to clean up my forked repo, but I've already spent too long trying to fix this xD

**Comment by Abraxas000** [comment]
New PR: https://github.com/PokemonGoF/PokemonGo-Bot/pull/2269
